.rubocop_todo.yml

```diff
@@ -0,0 +1,4 @@
+Style/HashSyntax:
+  EnforcedStyle: hash_rockets
+RSpec/NamedSubject:
+  Enabled: false
```

Comment:
Unsure how ok this is?
It used to be disabled and there are a serious amount of violations of this cop. It would result in some files being completely rewritten.

Reply:
I would say this is fine for now. Plus it won't get lost if it's in the todo, so seems perfect solution for the meantime.